### ISSUE TITLE
Enhancement: Add `frameStart` and `frameEnd` to template keys

### ISSUE THREAD

**Issue by RdMaple** [ynput/OpenPype#6009]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

I can't find any way to insert frame number range in template,so when some project  need have frame range in workfile ,can't do this use template.

### How would you imagine the implementation of the feature?

put frame range to keys maybe

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the enhancement request.

[cuID:[OP-7545](https://app.clickup.com/t/86bwq53nr)]

**Comment by MustafaJafar** [comment]
As far as I know, you can set frame range per project, per asset, per task 
per project: 
![image](https://github.com/ynput/OpenPype/assets/20871534/afbc1837-a4ce-4b17-94b9-bfbf789dbb51)
per asset and per task: 
![image](https://github.com/ynput/OpenPype/assets/20871534/ceae437a-6266-4429-9f26-51edff0d5b3d)

Is that what you are looking for ?

**Comment by MustafaJafar** [comment]
Yes, I can tell `frameStart` and `frameEnd` are not found in this list [available-template-keys](https://ayon.ynput.io/docs/admin_settings_project_anatomy/#available-template-keys)
